src/main/java/net/minecraftforge/client/event/ScreenEvent.java

```diff
@@ -233,6 +240,32 @@ public PotionShiftEvent(Screen screen)
         }
     }
 
+    /**
+     * Fired to determine whether to render the potion indicators in the {@link EffectRenderingInventoryScreen inventory
+     * screen} in compact or classic mode.
+     *
+     * <p>This event is not {@linkplain Cancelable cancelable} and {@linkplain HasResult has a result}. </p>
+     * <ul>
+     *   <li>{@link Result#ALLOW} - forcibly renders the potion indicators in <em>compact</em> mode.</li>
+     *   <li>{@link Result#DEFAULT} - defaults to vanilla behavior to using compact mode if the the screen width is too
+     *   small for classic rendering of potion indicators.</li>
+     *   <li>{@link Result#DENY} - forcibly renders the potion indicators in <em>classic</em> mode.</li>
+     * </ul>
+     *
+     * <p>This event is fired on the {@linkplain MinecraftForge#EVENT_BUS main Forge event bus},
+     * only on the {@linkplain LogicalSide#CLIENT logical client}. </p>
+     */
+    @HasResult
+    public static class PotionSizeEvent extends ScreenEvent {
+        /**
+         * @hidden For internal use only.
```

Comment:
Can this tag be compiled by the `javadoc` task? From what I remember, it would need to be added as a custom tag to the task configuration.

Reply:
This is a valid tag according to the [Documentation Comment Specification for the Standard Doclet (JDK 17)](https://docs.oracle.com/en/java/javase/17/docs/specs/javadoc/doc-comment-spec.html#hidden), validated by its existence at [`com.sun.source.doctree.HiddenTree`](https://github.com/openjdk/jdk17u/blob/master/src/jdk.compiler/share/classes/com/sun/source/doctree/HiddenTree.java):
> ### `@hidden`
>
> - `@hidden`
>
> Hides a program element from the generated API documentation. This tag may be used when it is not otherwise possible to design the API in a way that such items do not appear at all.
> 
> Introduced in JDK 9.